src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -4991,6 +4994,20 @@ void ObjectMgr::LoadQuests()
                 nextQuestItr->second.DependentPreviousQuests.push_back(qinfo->GetQuestId());
         }
 
+        if (uint32 breadcrumbForQuestId = std::abs(qinfo->_breadcrumbForQuestId))
+        {
+            auto targetQuestItr = _questTemplates.find(breadcrumbForQuestId);
+            if (targetQuestItr == _questTemplates.end())
+            {
+                TC_LOG_FATAL("sql.sql", "Quest %u is a breadcrumb for quest %u, but no such quest exists", qinfo->_id, breadcrumbForQuestId);
+                exit(1);
```

Comment:
please just gracefully handle the case and log only a TC_LOG_ERROR, there's not need to shutdown worldserver for just a quest

Reply:
This one can be handled gracefully.